cudapoa/include/claragenomics/cudapoa/batch.hpp

```diff
@@ -44,6 +44,43 @@ struct Entry
 /// A type defining the set and order of Entry's in which a POA is processed.
 typedef std::vector<Entry> Group;
 
+/// A structure to hold  upper limits for data size processed in POA batches
+struct UpperLimits
+{
```

Comment:
we can also add the `max_sequences_per_poa` into this struct (the first param to the `create_batch` call) and rename this struct more appropriately. I'll keep thinking on it :)

Reply:
yes, good idea. May be something like POASize, POADimensions, BatchSize, BatchDimensions...?

Reply:
added `max_sequences_per_poa` and changed struct name to `BatchSize`

Reply:
Thanks!